addons/Hierarchical_Table_Of_Contents/src/presenter.js

```diff
@@ -252,6 +253,13 @@ function AddonHierarchical_Table_Of_Contents_create() {
     };
 
     presenter.validateModel = function (model) {
+        var expandDepth = returnCorrectObject(0);
```

Comment:
W tym wypadku stylistyczne, ale zawsze staram się dawać const przy deklaracji zmiennych których referencja się nie zmienia. Myślę, że to spoko praktyka

Reply:
Faktycznie, mój błąd, umknęło mi to ponowne przypisanie :P